docs/getting-started/contribute.md

```diff
@@ -10,7 +10,7 @@ Kubeedge is Apache 2.0 licensed and accepts contributions via GitHub pull reques
 ## Getting started
 
 - Fork the repository on GitHub
-- Read the [setup](../setup/setup.md#build) for build instructions
+- Read the [setup](../setup/setup.md) for build instructions
```

Comment:
please leave the # attribute
this will specify an headline on the page which will be displayed on the top of your screen.

Reply:
- it's a compromise as stated in the commit message.
- there is no such an attribute either in html or md.

https://docs.kubeedge.io/en/latest/setup/setup.html#build
https://github.com/kubeedge/kubeedge/blob/master/docs/setup/setup.md#build

Reply:
yes your second point is the important one!
(Github can handle tags; On docs.kubeedge.io we can handle tags too)